src/Emitter.php

```diff
@@ -186,6 +186,14 @@ public function emitBatch(array $events)
         return $results;
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function dispatch(object $event)
+    {
+        return $this->emit($event);
```

Comment:
Doesn't `emit()` expect `$event` to be either a string or an `EventInterface` instance?

I would love to see `emit()` relaxed so that it can accept any object as an event, which is the intent of PSR-14.

Reply:
That is correct. I know there would need to be a guaranteed way to get an event's name/alias, maybe default to `get_class($event)`? Other event methods like `Event::isPropagationStopped()` and `Event::setEmitter()` can just be handled by using `Psr\EventDispatcher\StoppableEventInterface` and `EventAwareInterface`/`EventAwareTrait`, respectively.